src/Our.Umbraco.StackedContent/Web/Helpers/ViewHelper.cs

```diff
@@ -1,15 +1,36 @@
 ﻿using System.IO;
+using System.Linq;
 using System.Web;
 using System.Web.Mvc;
 using System.Web.Routing;
+using Umbraco.Core.Logging;
 
 namespace Our.Umbraco.StackedContent.Web.Helpers
 {
-    internal static class ViewHelper
+    public static class ViewHelper
     {
-        class DummyController : Controller { }
+        private class DummyController : Controller { }
 
-        public static string RenderPartial(string partialName, object model, string[] viewLocations)
+        private static readonly RazorViewEngine ViewEngine = new RazorViewEngine
+        {
+            PartialViewLocationFormats = new[]
+            {
+                "~/Views/Partials/Stack/{0}.cshtml",
+                "~/Views/Partials/Stack/Default.cshtml"
+            }
+        };
+
+        public static void AddViewLocations(params string[] viewLocations)
```

Comment:
Should this be `AddViewLocationFormats` in order to suggest these are format strings?

Reply:
Perhaps. Guess I kinda assumed people would need to know that anyways.